protocol-specs/generators/block_processing_votes/index.js

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	decryptPassphraseWithPassword,
+	parseEncryptedPassphrase,
+	getAddressFromPrivateKey,
+	getPrivateAndPublicKeyFromPassphrase,
+} = require('@liskhq/lisk-cryptography');
+
+const BaseGenerator = require('../base_generator');
+const defaultConfig = require('../../config/devnet');
```

Comment:
We need to discuss how much we want this tool to grow and allot time based on that as this issue is only for generating the JSON I'm trying to avoid big involvements in the library itself as that is just a tool and not the objective but I do agree that we might want to check things like this.

Reply:
Agree, thanks! Let's just keep track of these possible improvements.